test/service/test_request_validation.py

```diff
@@ -227,9 +227,9 @@ def test_bdbag_manifest(self):
                 samples = iter(csv.reader(fh, delimiter='\t'))
                 row = next(samples)
                 self.assertEqual(expected_fieldnames, row, 'Manifest headers are not configured correctly')
-                row = next(samples)
-                self.assertEqual(row, expected_row, 'Row in sample.tsv incorrect.')
-                remaining_rows = list(samples)
-                self.assertTrue(all(len(row) == len(expected_row) for row in remaining_rows),
+                samples = list(samples)
+                self.assertEqual(len(samples), 2, 'Found incorrect number of files.')  # self.bundle has 2 files
+                self.assertTrue(all(len(sample) == len(expected_row) for sample in samples),
                                 'Incorrect number of entries in a row.')
-                self.assertEqual(len(remaining_rows), 1, 'Found incorrect number of files.')  # self.bundle has 2 files
+                self.assertIn(expected_row, samples, 'Expected file was not found or had incorrect fields')
```

Comment:
Can you think of an easy way to establish that `expected_row` only occurs once in samples.

Reply:
Not without a loop or something. Let me know if you want me to make it like that.